modules/table/src/main/java/org/apache/ignite/internal/table/distributed/raft/snapshot/incoming/IncomingSnapshotCopier.java

```diff
@@ -393,7 +393,7 @@ private String createPartitionInfo() {
     private void writeVersion(ResponseEntry entry, int i) {
         RowId rowId = new RowId(partId(), entry.rowId());
 
-        HybridTimestamp timestamp = i < entry.timestamps().size() ? entry.timestamps().get(i) : null;
+        Long timestamp = i < entry.timestamps().length ? entry.timestamps()[i] : null;
```

Comment:
Can we move `i < entry.timestamps().length` check to the `if` below? What's the point of creating a `Long` here?